.make/check_gopath.go

```diff
@@ -11,36 +11,45 @@ import (
 	"strings"
 )
 
+func isModuleFlagOn() bool {
+	var moduleFlag = os.Getenv("GO111MODULE")
+	if moduleFlag == "on"  {
+		return true
+	}
+	return false
+}
+
 // Checks if the packageName is checked out into one of the GOPATH entries
 // under GOPATH[i]/src/packageName.
 func main() {
-	var packageName string
-	flag.StringVar(&packageName, "packageName", "github.com/fabric8-services/build-tool-detector", "Package Name (e.g.)")
-	flag.Parse()
+	if !isModuleFlagOn(){
```

Comment:
Why not just return if module support is used? Then you can get rid of the indentation.

```suggestion
	if isModuleFlagOn(){return}
```